_pages/help.md

```diff
@@ -2,3 +2,82 @@
 layout: help_landing
 permalink: /help/
 ---
+<article class="grid-container-tablet-lg tablet-lg:padding-x-0 margin-top-9 padding-bottom-1">
+  <ul class="usa-card-group grid-row tablet:flex-align-center usa-list usa-list--unstyled">
+    <li class="card">
+      <div class="grid-row flex-row tablet:flex-align-center">
+        <div class="grid-col-2">
+          <div class="usa-card__img">
+            <img alt=" " src="{{ site.baseurl }}/assets/img/help/get-started.svg">
```

Comment:
If this is meant to be [decorative](https://www.w3.org/WAI/tutorials/images/decorative/), then I think the alt needs to be...

```suggestion
            <img alt="" src="{{ site.baseurl }}/assets/img/help/get-started.svg">
```

Otherwise it appears to be interpreted literally:

![image](https://user-images.githubusercontent.com/1779930/100793449-19042d00-33ea-11eb-89ee-b60cbecc1342.png)

Same note for all the other images accompanying these cards.